src/components/inline/InputText/InputText.stories.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+import InputText from "./InputText";
+
+/*
+For responsive behavior, the width of the parent element
+of this component should be set in a class and should
+follow this example:
+
+In your js file:
+<span className="my-wrapper"><InputText></span>
+
+In your css file:
+.my-wrapper {
+  width: 400px;
+}
+
+@media (max-width: 720px) {
+  .my-wrapper {
+    width: 200px;
+  }
+}
+ */
+
+const demoStyling = {
+  left: "50%",
+  position: "absolute",
+  top: "50%",
+  transform: "translate(-50%, -50%)",
+  width: "200px",
+};
+
+export default {
+  title: "Inline/InputText",
+  component: InputText,
+};
+
+// Variables declared here so that DefaultInputText.args can interact with useState
+let text;
+let setValue;
```

Comment:
Wait, maybe it just doesn't make any sense to put `onChange` in the `args` at all? It's not like a user will be able to configure it from the Storybook UI anyway. Could you just remove that arg and define it inline in the Template body?

Reply:
yes

Reply:
Unresolving this because you don't need to declare these as mutable variables out in the global scope if the handleChange function is being defined inline now.